### DISCUSSION TITLE
How to disable feeds by category?

### DISCUSSION THREAD

**Discussion by mbnoimi** [FreshRSS/FreshRSS#4814] [Q&A] [answered]
I've many feeds under one category. How can I disable all these feeds from this category?
NOTE: I read [this old fix (I'm even not sure if it still valid)](https://github.com/FreshRSS/FreshRSS/issues/1036#issuecomment-159372905) but I wonder if there is UI solution instead of modifying the database manually.

**Comment by aledeg** [comment, marked as answer]
Currently, the only solution in the UI is to process them one by one.
If there is no open issue, you might create one.

**Reply by mbnoimi** [reply]
> Currently, the only solution in the UI is to process them one by one. If there is no open issue, you might create one.

Thanks. I opened a new feature request https://github.com/FreshRSS/FreshRSS/issues/4816
I wish to accept it because executing manual queries is really awful (I use SQLite db)
